### PR TITLE
Convex hull example

### PR DESCRIPTION
**Pull request by Vrixyz**
This builds on https://github.com/dimforge/parry/pull/250 to take advantage of `common_macroquad` code.

[Screencast from 09-03-2024 03:21:04 PM.webm](https://github.com/user-attachments/assets/d35e4502-c026-4e04-a6cc-ec056336d3a7)

[Screencast from 09-03-2024 04:06:20 PM.webm](https://github.com/user-attachments/assets/4abc89b5-4c53-4f59-ba2b-5af0e267dd78)

### CODE REVIEW

**Review by sebcrozet — COMMENTED** [review]
I think this (and all the similar PRs) needs to be rebased on `main` to lighten the size of te diff.

**Comment by Vrixyz** [comment]
I merged master in ; in order to be able to reuse the same conflict resolution on similar pr

File: src/transformation/to_trimesh/cuboid_to_trimesh.rs [hunk 1]
```diff
@@ -3,19 +3,14 @@ use crate::math::{Point, Real};
...
 
-#[cfg(feature = "dim3")]
-use na::Point3 as PointDim;
-#[cfg(feature = "dim3")]
-use na::{self, Point3};
-
 #[cfg(feature = "dim2")]
 use na::Point2 as PointDim;
 #[cfg(feature = "dim2")]
 use na::{self, Point2};
```
Comment (sebcrozet) [inline src/transformation/to_trimesh/cuboid_to_trimesh.rs]: Are these still needed?

**Review by sebcrozet — APPROVED** [review]
Thanks!

File: src/transformation/to_trimesh/cuboid_to_trimesh.rs (hunk 1 above)
Comment (Vrixyz) [reply src/transformation/to_trimesh/cuboid_to_trimesh.rs]: This was addressed in a previous PR :ok_hand:

**Pull request merged**